mmv1/third_party/terraform/resources/resource_dataproc_cluster.go.erb

```diff
@@ -13,11 +13,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 
-<% if version == "ga" -%>
```

Comment:
Can we remove all of the `<% if version == "ga" -%>` blocks in the dataproc resources?

If you don't have the capacity to do so, we can open an issue and do that separately, just wanted to mention that's our end goal. Thanks!

Reply:
I did some clean up, but there are still several version guards, I will create an issue and work on that after the holiday.